.gitattributes

```diff
@@ -1,55 +1,38 @@
 # Git config here
 * text eol=lf
-
```

Comment:
Why delete the newlines?    
They are there for readability

Reply:
I didn't do it, after I ran `git lfs add` and it formatted it

Reply:
Yeah, that command is good for the initial setup but after that to make it maintainable it has to be for humans.    
So would ye be able to revert yer changes on this file since ``.png`` is already covered by it.

Reply:
No longer in scope